src/features/People/peopleSaga.js

```diff
@@ -41,9 +44,20 @@ function* fetchPersonDetailsHandler({ payload: id }) {
     yield put(fetchPersonDetailsError());
   }
 }
+function* fetchPeopleByQueryHandler({ payload: query, page}) {
+  console.log("Ppl Saga: ", query, page);
+  try {
+    yield delay(500);
```

Comment:
I'd give delay(100), like in the movies 😃

Reply:
right, done :D